src/views/Stake/Stake.jsx

```diff
@@ -109,6 +110,15 @@ function Stake() {
       // eslint-disable-next-line no-alert
       dispatch(error("Please enter a value!"));
     } else {
+      // 1st catch if quantity > balance
+      let gweiValue = ethers.utils.parseUnits(quantity, "gwei");
+      if (action === "stake" && gweiValue.gt(ethers.utils.parseUnits(ohmBalance, "gwei"))) {
+        console.log("in stake ok");
+        return dispatch(error("You cannot stake more than your OHM balance."));
```

Comment:
A stylelistic thing:
I love `else-less` code if possible. You return here, so technically you don't need another `else if` underneath. 
Depending on the software you can also structure your code to do early returns, for errors (like you have here) and have the code flow normally.